src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -149,14 +143,14 @@ public Arm() {
         for (int i = 0; i < motors.length; i++) {
             // create the motor
             motors[i] = new TalonFX(ArmConstants.MOTOR_IDS[i]);
-            // TODO: why is this done here? The slave should follow when neutral mode is set below.
             motors[i].setNeutralMode(NeutralModeValue.Brake);
 
```

Comment:
OK. What is the answer to the TODO question?
If you have an answer, then it should be in a comment right here.
Do not delete questions that you cannot answer.

Reply:
We want to make sure all of the motors are braking, and I removed the setNeutralMode below so this is the only one. If this is overridden, it's 3 extra calls to a simple method.